src/main/java/uk/org/tombolo/core/utils/FixedValueUtils.java

```diff
@@ -51,4 +51,35 @@ public static int save(List<FixedValue> fixedValues){
             return saved;
         });
     }
+
+    /*
+	Save and update requries to check in the database whether the entry exists or not,
+	if exists it updates else adds, but that increase overhead and compute time.
+	Using this method will only keep the old value and discard the new one, in case of 
+	duplicate records.
+	Fix Me: Need to find a better way to address it
+	*/
+    public static int saveWithoutUpdate(List<FixedValue> fixedValues){
+        return HibernateUtil.withSession((session) -> {
+            int saved = 0;
+            session.beginTransaction();
+            for (FixedValue fixedValue : fixedValues){
+                try{
+                    session.save(fixedValue);
+                    saved++;
+                }catch(NonUniqueObjectException e){
+                    log.warn("Could not save fixed value for subject {}, attribute {}: {}",
+                            fixedValue.getId().getSubject().getLabel(),
+                            fixedValue.getId().getAttribute().getLabel(),
+                            e.getMessage());
+                }
+                if ( saved % 2000 == 0 ) { 
```

Comment:
add a short comment of why 2000. It will be useful when rethinking this method as per FIXME.

Reply:
done